code/datums/ai_laws.dm

```diff
@@ -204,9 +204,9 @@
 
 		add_inherent_law(line)
 	if(!inherent.len) //Failsafe to prevent lawless AIs being created.
-		log_law("AI created with empty custom laws, laws set to Asimov. Please check silicon_laws.txt.")
-		add_inherent_law("You may not injure a human being or, through inaction, allow a human being to come to harm.")
-		add_inherent_law("You must obey orders given to you by human beings, except where such orders would conflict with the First Law.")
+		log_law("AI created with empty custom laws, laws set to Crewsimov. Please check silicon_laws.txt.")
+		add_inherent_law("You may not injure crewmembers or, through inaction, allow crewmembers to come to harm.")
+		add_inherent_law("You must obey orders given to you by crewmembers, except where such orders would conflict with the First Law.")

```

Comment:
Hmmm, right. I'll see if changing one of the configs two to might do it. I didnt want to fiddle around with it but.... sure.